CHANGELOG.md

```diff
@@ -8,8 +8,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Added
+- conftest: `--elfs_dir` argument to specify elfs directories
+- conftest: `--all` argument to run tests on all devices
 
 ### Changed
+- conftest: Default to using elf file in `bin/app.elf` if `--elfs_dir` is not specified
+- conftest: Targeted device must be specified, in doesn't default to all by default anymore
```

Comment:
So by default it will run on `./bin/app.elf` ? So if pytest is ran from the project root or the test root we don't have the same behavior?

If we specify a directory we will look for `<dir>/<appname>_<device>.elf`
I'll be honest I find this really inconsistent and a regression to what we currently have :/